ui_src/src/components/filter/style.scss

```diff
@@ -0,0 +1,151 @@
+.ant-popover {
+    padding-top: 0;
+}
+
+.ant-popover-inner {
+    width: 280px !important;
+    border-radius: 8px;
+}
```

Comment:
This will affect all popovers. You need to wrap it with className.

Reply:
Fixed